pick.js

```diff
@@ -45,8 +47,9 @@ function copy (from, to) {
         }
```

Comment:
I think if keypather should support regexps then that would be a change in keypather :P

since you check to make sure it's actually a regular expression, there's no chance of keypather and regular expression colliding